src/pages/dashboard/game/[id].tsx

```diff
@@ -49,27 +47,28 @@ const GameDashboard: NextPage<Props> = ({ game }) => {
                 name="allow_nsfw"
                 label="Allow nsfw"
                 value={allowNsfw}
-                onChange={() => setAllowNsfw(old => !old)}
+                disabled={true}
+                onChange={() => toggleAllowNsfw()}
```

Comment:
We can just pass the function reference here, saves creating a new function each render

```suggestion
                onChange={toggleAllowNsfw}
```